Gems/ROS2/Code/Include/ROS2/Sensor/ROS2SensorComponent.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include "ROS2/ROS2GemUtilities.h"
+#include "SensorConfiguration.h"
+#include <AzCore/Component/Component.h>
```

Comment:
The ROS2GemUtilities.h include should use angle brackets: https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#including-headers
```suggestion
#include "SensorConfiguration.h"
#include <ROS2/ROS2GemUtilities.h>
#include <AzCore/Component/Component.h>
```